softwarereview_intro.pt.Rmd

```diff
@@ -73,6 +73,8 @@ source(file.path("scripts", "airtable-access.R"), local = knitr::knit_global())
 source(file.path("scripts", "airtable-get-reviewers.R"), local = knitr::knit_global())
 ```
 
+### Editores(as) Convidados(as) {#guesteditors}
```

Comment:
This is not in the original version (english). But to follow the structure, I added this h3-title. 

Let me know what do you think, Yani. We can also edit it in English and Spanish.